build-fail-reminder.py

```diff
@@ -242,7 +242,7 @@ def main(args):
                 # Package failed to build for 6 weeks - file a delete request
                 r = osc.core.Request()
                 r.add_action('delete', tgt_project=project, tgt_package=package)
-                r.description = "[botdel] Package has had build problems for &gt;= 6 weeks"
+                r.description = "[botdel] Package has had build problems for >= 6 weeks"
```

Comment:
```suggestion
                r.description = "[botdel] Package has failed to build for at least 6 weeks"
```

Reply:
We might even change the wording - as the bot also files del req for stuff 'failing to install' for 6 weeks (originally, the bot only did buildfail errors, but has evolved to also act on installcheck issues)

Reply:
It should probably also mention the reason in that case